### ISSUE TITLE
Provide reliable HTML snippet for embedding in blogs and news articles

### ISSUE THREAD

**Issue by lidel** [meandavejustice/pin-tweet-to-ipfs#126]
## Problem

News articles, blogposts, and mirrors still hotlink twitter posts. 
They are loaded dynamically and if twitter.com removes/censors them, all the context is lost.

### Example

| with JS fetched from twitter.com | without JS from twitter.com | 
| ---- | ---- |
| ![threadreaderapp com_thread_1615748869723394049 html (1)](https://user-images.githubusercontent.com/157609/213932213-d0dd1278-43b2-4109-95ee-905f694912be.png)   | ![Screenshot 2023-01-22 at 18-59-59 Thread by @kyliebytes on Thread Reader App](https://user-images.githubusercontent.com/157609/213932076-e9b99e6c-6fe2-49ed-b97e-db6f1b72307f.png) | 

## Proposed Solution

Extend user interface  displayed to the user after "Tweet is Pinned!":

>  ![2023-01-22-190946_731x444_scrot](https://user-images.githubusercontent.com/157609/213932502-fe56e753-c670-435d-ba3b-5a830a5d2fe9.png)

In addition to the already present shareable link to content on gateway, have HTML with `<script>` or `<iframe>` that can be copied and pasted to embedd mirrored tweet.

### Technical challenges 

Note that this may be technically tricky to pull off, since Service Worker used by Webrecorder may not work in cross-origin `<iframe>`, but could be done with `<script>`. `<script>` could be made robust, retrywith [a public gateway](https://ipfs.github.io/public-gateway-checker/src/gateways.json)  if default gateway is down, fetch content as CAR and verify it before rendering etc.

**Comment by ikreymer** [comment]
Yes, unfortunately, due to the service-worker based nature of the replay, there's not a guaranteed HTML snippet that someone can add, without also hosting a local service worker that imports from the gateway locally. We could still show all of that, its just one more step than just a snippet, currently.

Embedding an iframe will also not work, as service workers are now being blocked on third party iframes along with third party cookies (Chrome doesn't have it on by default by FF and Safari do).
As a result, the service worker won't load in an <iframe> most likely unless its on a subdomain..

This is indeed one limitation of the current system.
